### ISSUE TITLE
[FBMN] Workshop feedback - In which file was that compound annotated ?!

### ISSUE THREAD

**Issue by lfnothias** [CCMS-UCSD/GNPS_Workflows#526]
Currently, the only way to know in which file a compound was annotated, is very cumbersome:
1) Either going into Cytoscape, assuming the metadata table has one group per sample
2) Going back to MZmine or processing and manually inspecting.

I am not sure how this can tackle reasonably ...
For classical molecular networking we have a column with the list of filenames, maybe that can be done from the quant table ?!

**Comment by mwang87** [comment]
Yeah, you lose this information in the quant table because you have no idea where the MS2 came from, just where it was most abundant.

**Comment by lfnothias** [comment]
Can't the quant table be used for that instead ? I understand the challenge with MS1 intensity and noise level ...
